Sources/hostmgr/commands/sync/SyncAuthorizedKeysCommand.swift

```diff
@@ -4,35 +4,16 @@ import libhostmgr
 
 struct SyncAuthorizedKeysCommand: AsyncParsableCommand, FollowsCommandPolicies {
 
+    struct Constants {
+        static let s3Key = "authorized_keys"
+        static let destination = Configuration.shared.localAuthorizedKeys
+    }
```

Comment:
Nitpick. I once saw this kind of information container type being an `enum`, which I found neat because it prevents doing `_ = Constants()`.

Reply:
Neat! Done in e7eabdbb01d322ef4bd2caff2d21ac3bd8910b3d